src/partials/forms/forms.component.ts

```diff
@@ -75,6 +82,10 @@ export class FormsComponent implements OnInit {
           if (res.err) {
             console.error(res.err);
             return;
+          } else {
+            let successTime = 3000;
```

Comment:
move it to global for this module constant value(not declare it 3 times)
Maybe, you should move all code which work with `success` state to function, like this
```
setPageSuccess(status) {
  this.success = status;
  if (status) {
    setTimeout(() => this.success = false, successTime);
  }
}
```

my point is: this way you can explicitly manipulate `success` state of page from one place of code